docs/includes/google-credentials.md

```diff
@@ -1,24 +1,28 @@
-**Tip:** If you see Google Play Console or Google Developer Console in your local language, add `&hl=en` at the end of the URL (before any `#...`) to switch to English.
+**Tip:** If you see Google Play Console or Google Developer Console in your local language, add `&hl=en` at the end of the URL (before any `#...`) to switch to English. All the links below already have this to make it easier to find the correct buttons.
 
-1. Open the [Google Play Console](https://play.google.com/console/)
+1. Open the [Google Play Console](https://play.google.com/console/?hl=en)
 1. Click **Account Details**, and note the **Developer Account ID** listed there
-1. Click **Setup** → **API access**
-1. Click the **Create new service account** button
-1. Follow the **Google Cloud Platform** link in the dialog, which opens a new tab/window:
+1. Open [Create Project on Google Cloud](https://console.cloud.google.com/projectcreate/?hl=en) and create a project if none exists that fits
+1. Enable the [Google Play Developer API](https://console.developers.google.com/apis/api/androidpublisher.googleapis.com/?hl=en) by selecting the project and pushing "enable"
+1. Open [Service Accounts on Google Cloud](https://console.cloud.google.com/iam-admin/serviceaccounts?hl=en) and select the project you used before
    1. Click the **CREATE SERVICE ACCOUNT** button at the top of the **Google Cloud Platform Console**
-   1. Verify that you are on the correct Google Cloud Platform Project by looking for the **Developer Account ID** from earlier within the light gray text in the second input, preceding `.iam.gserviceaccount.com`. If not, open the picker in the top navigation bar, and find the one with the **ID** that contains it.
-   1. Provide a `Service account name` and click **Create**
+   1. Verify that you are on the correct Google Cloud Platform Project by looking for the **Developer Account ID** from earlier within the light gray text in the second input, preceding `.iam.gserviceaccount.com`. If not, open the picker in the top navigation bar, and find the one with the **ID** that contains it
+   1. Provide a `Service account name` (e.g. fastlane-supply) and click **Create**
+   1. Copy the generated E-Mail address that is noted below the ServiceAccount-ID field for later use
    1. Click **Select a role**, then find and select **Service Account User**, and proceed
```

Comment:
We have reports (https://github.com/fastlane/docs/issues/1166) stating that this user doesn't need any role, apparently. I wasn't able to confirm this personally yet, but I think a friend of mine also observed this (although he can't tell for sure). Can anyone confirm whether we can create an account in Google Cloud Console without any role, and then configure the Service Account User all the permission it needs via Google Play Console? 👀 that would be waaay better